wsmaster/codenvy-hosted-system-ram/src/main/java/com/codenvy/service/systemRam/SystemRamLimitChecker.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.service.systemRam;
+
+/**
+ * Checks systems ram for 90% free ram existence.
+ *
+ * @author Igor Vinokur
+ */
+public interface SystemRamLimitChecker {
+
+    String getSystemRamLimitStatus();
```

Comment:
Why `String`, it looks like this interface should return an object describing system ram status, or at least an enum, 
i see the component "like this" pretty useful and i think it can be reused in other places as well, but the method declaration it is not clear _what should be returned_ ?